Test/git-issues/git-issue-889a.dfy

```diff
@@ -0,0 +1,128 @@
+// RUN: %dafny "%s" > "%t"
```

Comment:
Check --128, with and without space, (for both bv8 and an 8-bit new type), just for good measure

Reply:
Done